### ISSUE TITLE
Is there a function to reverse the output?

### ISSUE THREAD

**Issue by adelriosantiago** [segmentio/xml-parser#26]
**Is there a function that "reverses" from the parsed output back to XML?**

I'm looking to parse, do some changes in the parsed output and then bring it back to XML.

**Comment by christianroy** [comment]
> **Is there a function that "reverses" from the parsed output back to XML?**

No, afaik (and can tell from the source) there is no such function.

**Comment by mk-pmb** [comment]
The kind of module you're looking for is called "xml writer". npm has plenty to choose from. Maybe you can find one whose in-memory data format is compatible.
